### PR TITLE
Add auth notice to installation instructions

### PR DESCRIPTION
**Pull request by luap42**
### Pull Request summary

This PR adds a notice to the installation instructions, that one also needs to set up auth.

### Pull Request Tasklist

- [ ] Pull request references an issue
- [ ] Branch follows contributing guidelines format
- [x] Pull request targets the `develop` branch

### CODE REVIEW

File: docs/linux-instructions.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ shortcuts.
  3. You can launch the application by executing `dotnet run` in the `src/WebUI` directory.
 
  4. You can run the tests by executing `dotnet test` in the root directory.
+ 
+ ## Setting up the auth provider
+ 
+ You'll also need to set up the auth provider. See [this instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```
Comment (asynts) [inline docs/linux-instructions.md]: I am not a native speaker, is "those instructions" correct?
```suggestion
You will also need to set up the authentication provider. See [those instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```

File: docs/windows-instructions.md [hunk 2]
```diff
@@ -68,6 +68,11 @@
  5. You can launch the appliation by executing `dotnet run` in the `src/WebUI` directory.
 
  6. You can run the integration tests using `dotnet test` in the root directory.
+ 
+  
+ ## Setting up the auth provider
+ 
+ You'll also need to set up the auth provider. See [this instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```
Comment (asynts) [inline docs/windows-instructions.md]: Same as above.
```suggestion
You will also need to set up the authentication provider. See [those instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```

File: docs/linux-instructions.md [hunk 3]
```diff
@@ -88,6 +88,10 @@ shortcuts.
  3. You can launch the application by executing `dotnet run` in the `src/WebUI` directory.
 
  4. You can run the tests by executing `dotnet test` in the root directory.
+ 
+ ## Setting up the auth provider
```
Comment (asynts) [inline docs/linux-instructions.md]:
```suggestion
 ## Setting Up The Authentication Provider
```

File: docs/windows-instructions.md [hunk 4]
```diff
@@ -68,6 +68,11 @@
  5. You can launch the appliation by executing `dotnet run` in the `src/WebUI` directory.
 
  6. You can run the integration tests using `dotnet test` in the root directory.
+ 
+  
+ ## Setting up the auth provider
```
Comment (asynts) [inline docs/windows-instructions.md]:
```suggestion
 ## Setting Up The Authentication Provider
```

**Review by asynts — APPROVED** [review]
Looks good. Treat these suggestions with care because I am not a native speaker.

File: docs/linux-instructions.md (hunk 1 above)
Comment (misha130) [reply docs/linux-instructions.md]: Isnt it these instructions?

Comment (asynts) [reply docs/linux-instructions.md]: I just read an article about those *versus* these and I still do not know.

Comment (luap42) [reply docs/linux-instructions.md]: I am not a native speaker either, but "these" sounds more correct IMO. I'm going to make this change and then merge.

File: docs/linux-instructions.md (hunk 3 above)
Comment (luap42) [reply docs/linux-instructions.md]: I'll not change "up the" for consistency with the other headings.

**Pull request merged**